scripts/geojson/index.js

```diff
@@ -0,0 +1,134 @@
+const fs = require('fs'),
+      argv = require('minimist')(process.argv.slice(2)),
+      request = require('request'),
+      //extract = require('extract-zip'),
+      AdmZip = require('adm-zip'),
+      xmldom = new (require('xmldom').DOMParser)(),
+      tj = require('togeojson'),
+      mkdirp = require('mkdirp'),
+      path = require('path'),
+      { URL } = require('url');
+
+(function main(argv) {
+
+  const countryCode = argv._[0]
+  console.info(`# Starting ... ${countryCode}`)
+
+  const config = {
+    path: {
+      out: argv._[1] || '/output',
+      tmp: argv._[2] || '/tmp'
+    },
+    url: {
+      base: 'http://biogeo.ucdavis.edu/data/gadm2.8/kmz/'
```

Comment:
Is this URL source stable?

Reply:
yes, as stable as it can get for scripting.
during this month gadm3 should become available AFAIK 🤷‍♂️